docs/api.md

```diff
@@ -2211,6 +2198,76 @@ Verify a session using a recovery code.
   <!--end-response-body-post-sessionverifyrecoverycode-remaining-->
 
 
+### Recovery keys
+
+#### POST /recoveryKeys
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-post-recoverykeys-->
+Creates a new recovery key for a user. These are one time use
+keys.
```

Comment:
Seeing this here, it feels like we probably need a new document at `./docs/recovery_keys.md` that explains this mechanism in more detail, perhaps cribbed from the existing google doc.  Something in the style of https://github.com/mozilla/fxa-auth-server/blob/feature.pushbox-ng/docs/device_registration.md would be ideal (and I am happy to help write it).

Reply:
Let's link to your `docs/recovery_keys.md` doc from somewhere here.